core/shared/src/main/scala-2/org/typelevel/twiddles/TwiddleCompat.scala

```diff
@@ -115,7 +116,75 @@ trait TwiddleCompat {
   ): A *: B *: C *: D *: E *: F *: G *: H *: I *: J *: K *: L *: M *: N *: O *: P *: Q *: R *: S *:
     T *: U *: V *: EmptyTuple = t.productElements
 
-  implicit def hlistToTuple[L <: Tuple, T](l: L)(implicit
```

Comment:
If we leave this definition in place and simply remove the `implicit` on `implicit def`, we should be able to retain binary compatibility.